openapi/nexus.json

```diff
@@ -7261,6 +7257,21 @@
           }
         ]
       },
+      "ExternalIpList": {
+        "description": "Wrapper for unpaginated lists of items",
```

Comment:
Ew. Here's a reason to give each one its own struct.

Reply:
FWIW ResultsPage has a similar issue

Reply:
😕

Reply:
… that we could certainly use our cleverness to improve!!